<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>610</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>570</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Heavy</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Poligon/Resources/kisspng-complete-graph-laplacian-matrix-vertex-polygon-medical2-1629-free-pharmaceutical-medical-lo-5c9a165b8d58d9.177007451553602139579.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>431</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>230</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/button/Resources/kisspng-waon-fujita-kanko-inc-triangle-0-fast-forward-5b30268e02ab62.794417581529882254011.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>610</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>230</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/button/Resources/kisspng-waon-fujita-kanko-inc-triangle-0-fast-forward-5b30268e02ab62.794417581529882254011.png);
border-image: url(:/button_ters/Resources/TERS.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Franklin Gothic Heavy</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 170, 0);</string>
       </property>
       <property name="text">
        <string>ÖTELEME MİKTARI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Franklin Gothic Heavy</family>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(85, 255, 127);
color: rgb(6, 163, 79);</string>
       </property>
       <property name="text">
        <string>POLİGON ÇİZ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Franklin Gothic Heavy</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 85, 255);</string>
       </property>
       <property name="text">
        <string>YARIÇAP</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1288</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDonusumler">
    <property name="title">
     <string>Donusumler</string>
    </property>
    <addaction name="actionOlceklendir"/>
    <addaction name="actionOtele"/>
   </widget>
   <addaction name="menuDonusumler"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOlceklendir">
   <property name="text">
    <string>Olceklendir</string>
   </property>
  </action>
  <action name="actionOtele">
   <property name="text">
    <string>Otele</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
